Sources/Commands/SwiftBuildTool.swift

```diff
@@ -37,7 +37,8 @@ public class SwiftBuildTool: SwiftTool<BuildToolOptions> {
             toolName: "build",
             usage: "[options]",
             overview: "Build sources into binary products",
-            args: args
+            args: args,
+            seeAlso: "swift run, swift package, swift test"
```

Comment:
I think it would be better if we can make this typed. Maybe define a tool name protocol and conform all four tools to it?

Reply:
```swift
protocol ToolName {
    static var toolName: String
}

extension SwiftBuildTool: ToolName {
    static var toolName: String {
        return "swift build"
    }
}

...
seeAlso: [SwiftBuildTool.self, SwiftPackageTool.self, SwiftTestTool.self].map({ $0.toolName }).joined(separator: ",")
...


```

Reply:
I've made a function (`otherToolNames`) which can be called from any tool that conforms to `ToolName` to generate a string of other available tools (excluding itself). This way if new tools are added the list only needs to be updated in one place. Please let me know if this makes sense.